wikiconv/conversation_reconstruction/construct_utils/conversation_constructor_test.py

```diff
@@ -0,0 +1,116 @@
+"""Tests for ReconstructConversation."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import unittest
+
+from construct_utils import conversation_constructor
+
+
+class ConversationContructorTest(unittest.TestCase):
+
+  def test_conversation_constructor(self):
+    processor = conversation_constructor.Conversation_Constructor()
+    page_state = []
+    latest_content = []
+    rev = {
+        "comment": "*",
+        "sha1": "cb40d5f5b69538788c2f726822565f9a53e4452c",
+        "user_id": 8083618,
+        "format": "placeholder",
+        "user_text": "Anders Torlind",
+        "timestamp": "2001-09-11T18:22:34Z",
+        "contentformat": "text/x-wiki",
+        "user_ip": "placeholder",
+        "contentmodel": "wikitext",
+        "text":
+            "YIKES!  I came here from the [[Prostitution]] page - surely we "
```

Comment:
The text here is a little painfully long for a test, maybe shorten it to make the test easier to read?